cli/src/klio_cli/commands/job/gke.py

```diff
@@ -27,6 +27,12 @@
 
 
 class GKECommandMixin(object):
+    GKE_XPN_PROJECT = "gke-xpn-1"
+    GKE_UI_LINK_FORMAT = (
+        "https://console.cloud.google.com/kubernetes"
+        "/deployment/{region}/{cluster}/{namespace}/{app}"
+        "/overview?project=gke-xpn-1"
```

Comment:
Is this `project=gke-xpn-1` spotify specific or is that how all general users (including open source users) would access the k8s console?

Reply:
Ah, good catch - I changed it to extract the project from the context rather than hardcoding it.